### DISCUSSION TITLE
How to Update to the Latest Version of Ollama in Open-WebUI

### DISCUSSION THREAD

**Discussion by Jenosize** [open-webui/open-webui#3658] [General]
<img width="768" alt="Screenshot 2567-07-05 at 22 28 41" src="https://github.com/open-webui/open-webui/assets/138196283/84aebb78-a750-4859-af3b-a1e471ed1bbd">

Hi everyone,

I recently downloaded the latest version of Ollama (version 0.1.48). However, in Open-WebUI, it's still showing version 0.1.45.

Here's what I'm seeing:

Ollama version: 0.1.45
Warning: client version is 0.1.48
Does anyone know how to update Open-WebUI to use the latest Ollama version?

Thanks!

**Comment by igbon** [comment]
Hi, I have the same question. This is pretty typical of things that aren't Windows.

**Comment by Martin-Fox** [comment]
To update Ollama within Open-WebUI:
1. `docker exec open-webui apt-get update`
2. `docker exec open-webui curl -fsSL https://ollama.com/install.sh | sh` 
3. `docker restart open-webui`

Done :)
